lib/loader.js

```diff
@@ -271,6 +273,20 @@ module.exports = function (content) {
       '__vue_options__.staticRenderFns = __vue_template__.staticRenderFns\n'
   }
 
+  // add requires for customBlocks
+  if (parts.customBlocks && parts.customBlocks.length) {
```

Comment:
We should avoid adding any code to the file unless there is a matching loader for the custom block.